data/Krishnapro.json

```diff
@@ -1,8 +1,18 @@
 {
   "name": "Krishna Kumar",
+  "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Open Source & Cybersecurity Enthusiasts",
+  "bio": "Software Developer | Open Sorce | DevOps Enthusiasts",
```

Comment:
You have one minor spelling error here you may want to adjust.  Open Sorce --> Open Source

Otherwise LGTM 🥳

Reply:
@amandamartin-dev  hey I fixed it, please check